packages/backend/src/api/resources/index.ts

```diff
@@ -14,9 +14,10 @@ export type {
   OrganizationInvitationStatus,
   OrganizationMembershipRole,
   SignInStatus,
-  SignUpStatus,
 } from './Enums';
 
+export type { SignUpStatus } from '@clerk/types';
```

Comment:
Is this re-export needed?

Reply:
@jacekradko In theory, no, but I added it to keep consistency with the code above:

```
  SignInStatus,
} from './Enums';
```